package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "tape test/*.test.js",
-    "start": "node ./server.js",
+    "start": "node ./redesign/simple_server.js",
```

Comment:
merging this PR into master will "_break_" the current website...
![image](https://user-images.githubusercontent.com/194400/26931350-22319f8e-4c58-11e7-989a-4f0e53b54519.png)

@Cleop could you _instead_ update the `dev` command to:
```
"dev": "live-server --port=8000 --open=./redesign",
```
Then use `npm run dev` on your localhost while working on the new site's `index.html`